sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/QueryFeature.java

```diff
@@ -15,5 +15,6 @@ public enum QueryFeature {
     Top,
     NonValueAggregate,
     DCount,
-    NonStreamingOrderBy
+    NonStreamingOrderBy,
+    HybridSearch
```

Comment:
I assuem the query team made sure this time that Gateway would not return 400 if an unknown (to the service) query feature is announced by client? If not using latest SDK in air gapped clouds could cause issues?

Reply:
Yes, the HybridSearch query feature only passed in the query plan retriever when non streaming order by property is not disabled, similar to what we did for Vector Search.